### ISSUE TITLE
Holding strategy

### ISSUE THREAD

**Issue by arthurwolf** [c9s/bbgo#425]
I'm looking for a strategy that does the following:

* For pair XXX-YYY:
* At the beginning, buy as much XXX as possible using the YYY available
* Wait, do nothing, except checking prices compared to (configurable) stop-loss value. If price does below stop-loss for more than W time, sell all XXX for YYY back to prevent further loss.
* At the end, (if not done before because of stop-loss), sell all XXX for YYY.

This would be extremely helpful as a "baseline" strategy to compare other strategies to.

Is this currently possible using one of the strategies, maybe by using a specific set of configurations?
If not, I'd be interested in paying for implementation and documentation.

Thanks!

**Comment by c9s** [comment]
@arthurwolf  Yes, I designed a smartStop configuration, which will be integrated into different strategies later.

**Comment by arthurwolf** [comment]
(really cool!)
is this documented? how could I test/use this?

**Comment by c9s** [comment]
I will add the doc after the integration is done~

Arthur Wolf ***@***.***>於 2022年3月25日 週五，17:52寫道：
